### DISCUSSION TITLE
Can we use <UseMaui>true</UseMaui> in .NET Android application ?

### DISCUSSION THREAD

**Discussion by SarthakB26** [dotnet/maui#14285] [Q&A] [answered]
Hi, I am working on one .NET Android application due to some specific reasons. I Wanted to know if I can add <UseMaui>true</UseMaui> in my project csproj. I tried this but after adding maui my project no longer show any android device or simulator to deploy on also unable to use any maui specific class let say Connecivity class.

I am trying this to use maui.essentials classes. Otherwise I have to write those code on my own or installing third party nugets.

Has anybody tries this ?

**Comment by SarthakB26** [comment]
This is my application type

<img width="763" alt="Android" src="https://user-images.githubusercontent.com/125443207/228789318-08b6f4e1-f254-4334-b914-fa9526ea32a9.png">

**Reply by wollfoDream** [reply]
Same problem here.

**Comment by Dreamescaper** [comment]
You can probably use <UseMauiEssentials>true</UseMauiEssentials> if you don't need the whole MAUI.

**Comment by giuseppenovielli** [comment]
https://learn.microsoft.com/en-us/dotnet/maui/platform-integration/native-embedding

**Comment by wollfoDream** [comment, marked as answer]
The problem is when we set <UseMaui>True</UseMaui> in extra native project e.g. Android (as it was in Xamarin Forms .Droid) then in the Start button, device list is gone and if you press Start it will by default just pick last chosen device. Because of this we wonder if it should be done in another way or just Visual studio has a bug

![image](https://github.com/dotnet/maui/assets/23324424/e00cd89b-cf35-431f-a10c-9460234d8fbb)

**Comment by assurasc** [comment]
The marked answer does not solve the issue but just re-iterates it.
I tried both VS 17.6 and preview 17.7 and both have the same issue - adding `<UseMaui>True</UseMaui>` hides both emulators and devices in the start menu. I want to use Maui embedding in my .NET Android  application and cannot if I can't add `<UseMaui>`.

**Reply by SarthakB26** [reply]
what happen when you click on start ?

**Reply by SarthakB26** [reply]
@Redth  might know some workaround

**Reply by assurasc** [reply]
I raised an issue in this repo [here](https://github.com/dotnet/maui/issues/15388). The response is that this is an issue with Visual Studio and I should raise an issue with them which I will. There were some useful tips around running via command line and that it works correctly on VS Mac.

**Reply by assurasc** [reply]
Strangely enough, Its a @Redth library I'm trying to embed! zxing.net.maui. I might reach out to see if there is a way I can use it that does not require embedding.

**Comment by Sevenish** [comment]
Fixed in the current Preview version 
https://developercommunity.visualstudio.com/t/Adding-UseMaui-to-NET-Android-app-hides/10386552
